API/ev3_constants.h

```diff
@@ -45,8 +45,6 @@ extern "C" {
 /** @addtogroup MiscConstants
  * @{
  */
-#define TRUE  true /*!< A true value */

```

Comment:
I would leave those definitions in for backwards compatibility.

Reply:
Yea this might indeed break some user code. I'll leave it in.

Reply:
Is in again